data/assets/scene/solarsystem/sun/heliosphere/mas/bastille_day/density_volume.asset

```diff
@@ -0,0 +1,49 @@
+asset.info = {
```

Comment:
I originally thought we would place the info struct at the bottom of the file, but  think i like the top one better :)